Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Controls/ChangeDensityButton.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Xamarin.Forms.Controls
+{
+	public class ChangeDensityButton : Button
```

Comment:
While it's not wrong, I don't really see the need to introduce new controls for these kinds of things when you're only setting a couple of properties. Goes for the button and the label. If we do that for each issue then we will have a whole lot of new controls :)